From a0748b2698834005ff3bef8e0c557a2154ea5034 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Fri, 9 Mar 2018 07:13:55 +0100
Subject: [PATCH 62/74] libfm_jni: Resolve FM_DEVICE_PATH R/O open flag

 * libfm_jni/FmRadioController.cpp:96:58:
    error: 'open' has superfluous mode bits; missing O_CREAT?

Change-Id: I3f4f3bf4e27ea55539ec81b2e59b17be6dcb0892
---
 libfm_jni/FmRadioController.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libfm_jni/FmRadioController.cpp b/libfm_jni/FmRadioController.cpp
index a002121..ffcfb32 100644
--- a/libfm_jni/FmRadioController.cpp
+++ b/libfm_jni/FmRadioController.cpp
@@ -93,7 +93,7 @@ int FmRadioController ::open_dev()
 {
     int ret = FM_SUCCESS;
 
-    fd_driver = open(FM_DEVICE_PATH, O_RDONLY, O_NONBLOCK);
+    fd_driver = open(FM_DEVICE_PATH, O_RDONLY, 0);
 
     if (fd_driver < 0) {
         ALOGE("%s failed, [fd=%d] %s\n", __func__, fd_driver, FM_DEVICE_PATH);
-- 
2.17.1

